shared/actions/chat2/index.js

```diff
@@ -1284,10 +1285,10 @@ const changeSelectedConversation = (
           Saga.put(
             Chat2Gen.createSelectConversation({
               conversationIDKey: Constants.pendingConversationIDKey,
-              reason: 'searching',
+              reason: 'setPendingMode',
             })
           ),
-          ...(isMobile ? [Saga.put(Chat2Gen.createNavigateToThread())] : []),
```

Comment:
I don't think this test is correct -- if anything, it should be `!isMobile`, since on mobile select does a navigate.